code/modules/admin/holder2.dm

```diff
@@ -181,10 +181,14 @@ GLOBAL_PROTECT(href_token)
 
 	if(!feedback_query.Execute())
 		log_sql("Error retrieving feedback link for [src]")
+		qdel(feedback_query)
 		return cached_forum_link
 	if(!feedback_query.NextRow())
+		qdel(feedback_query)
 		return FALSE // no feedback link exists
 
+	qdel(feedback_query)
```

Comment:
Ugh, this broke feedback links because `feedback_query.item[1]` is null after this.

I wish these were easier to unit test without setting up a database. There's a lot we can do for this case specifically, but it is out of reason for anyone to do without being extremely dedicated to it

Reply:
Insert large numbers of a characters here.